Resources/Prototypes/Entities/Clothing/OuterClothing/vests.yml

```
@@ -28,6 +28,11 @@
       - state: equipped-OUTERCLOTHING-unshaded
         shader: shaded
         #same trick here
+  - type: Reflect
+    reflectProb: 0.05
+    reflects:
+      - Energy
```

```suggestion
    reflectProb: 0.02
    reflects:
    - Energy
```
I brought it up in maintainer chat again, and this should be more an easter egg rather than a regular thing, so I'm changing it to 2% like previously suggested.
While the 5% chance can be debated, we would rather avoid microbalancing these things.
And it's more fun when it happens if it is rare, similar to the wilhelm scream.